package.json

```diff
@@ -36,6 +36,8 @@
   "dependencies": {
     "@google-cloud/bigquery": "^5.0.0",
     "@google-cloud/local-auth": "^1.0.0",
+    "@types/backoff": "^2.5.1",
```

Comment:
It doesn't look like these are used.  Could I trouble you to remove them, and run `npm install` again to update the lock file?

Reply:
Yep, legacy from before you set me straight on retries.